content/get-started/quickstart/hello-world.md

```diff
@@ -89,8 +89,8 @@ You can make and save changes to the files in your repository. On {% data variab
 1. In the editor, write a bit about yourself. Try using different Markdown elements.
 {% ifversion code-search-code-view %}
```

Comment:
Hi @Shikha1234, I think we can simplify the versioning here to make it easier for people to see the logic

Can you modify lines 90-97 to the following?

```
{% ifversion code-search-code-view %}1. Click **Commit changes...**.{% endif %}
1. In the "Commit changes" box, write a commit message that describes your changes.
1. Click **Commit changes**.
```

Reply:
Is it the case that the lines:
`1. In the "Propose changes" box, write a commit message that describes your changes.
1. Click **Propose changes**.`
were left in erroneously - I noticed this issue too and glad to see someone was working on a fix! I was struggling to see why there were two instances of the same message!

Reply:
@callumjohnspiller, that's right, probably we changed the UI text to "Propose changes" at a certain point in the code search feature's development, then changed it back to "Commit changes" later